src/tif_model_iterator/tif_kernel_iterator.py

```diff
@@ -370,10 +428,10 @@ def func_cor_square(input_x_y, size: float = 2.0):
 
     # Not sure if dividing by 2 is the correct option.
     rect = [
-        input_x_y[0] - (size / 2.0),
-        input_x_y[1] - (size / 2.0),
-        input_x_y[0] + (size / 2.0),
-        input_x_y[1] + (size / 2.0),
+        input_x_y[0] - (2.5 / 2.0),
```

Comment:
Suggestion: This seems like a weird change? But more importantly, I believe fun_cor_square is not being used at all, so I would recommend just removing the entire function to avoid confusion!